lib/person.rb

```diff
@@ -0,0 +1,13 @@
+class Person
+  attr_accessor :book_shelf
+
+  def initialize
+    @book_shelf = []
+  end
+
+  def get_book(library, args={})
+    book = library.books.detect{|obj| obj[args.keys.first] == (args[args.keys.first]).to_s }
```

Comment:
Metrics/LineLength: Line is too long. [92/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)